Zend/zend_interfaces.stub.php

```diff
@@ -48,11 +48,11 @@ public function offsetUnset(mixed $offset);
 
 interface Serializable
 {
-    /** @return string */
+    /** @return string|null */
     public function serialize();
 
     /** @return void */
-    public function unserialize(string $data);
+    public function unserialize(?string $data);
```

Comment:
```suggestion
    public function unserialize(string $data);
```

Reply:
Commiting from GitHub won't work. You should update the hash in the arginfo file as well (ideally by running gen_stub.php).